src/main/java/io/confluent/idesidecar/restapi/application/SidecarInfo.java

```diff
@@ -62,7 +63,10 @@ static String semanticVersionWithin(Properties props, String key, String def) {
   static final String OS_ARCH_KEY = "os.arch";
   static final String OS_NAME_KEY = "os.name";
   static final String OS_VERSION_KEY = "os.version";
-  static final String SIDECAR_VERSION_KEY = "quarkus.application.version";
+  static final String SIDECAR_VERSION_KEY = ConfigProvider
```

Comment:
```suggestion
  static final String SIDECAR_VERSION = ConfigProvider
```

Also, can you remove the attribute `sidecarVersion` and let the method `version()` return the constant variable `SIDECAR_VARIABLE`?

Reply:
This breaks the tests:

```
      assertEquals(input.osVersion, sidecar.osVersion());
```

I'm torn between removing this from the tests and leaving it in... in case another part of the codebase will eventually rely on this attribute

Reply:
Hm, I don't think it will break this assertion but you need to update some other assertions because we can no longer pass a static sidecar version in the tests. Let's discuss it offline.